utils/soca/gdas_ens_handler.h

```diff
@@ -136,19 +132,58 @@ namespace gdasapp {
       // the ensemble mean and the deterministic
       soca::Increment recenteringIncr(geom, postProcIncr.socaIncrVar_, postProcIncr.dt_);
       recenteringIncr.diff(determTraj, ensMeanTraj);
-      eckit::LocalConfiguration sshRecErrOutputConfig(fullConfig, "ssh output.recentering error");
-      recenteringIncr = postProcIncr.setToZero(recenteringIncr);
+      postProcIncr.setToZero(recenteringIncr);
+
+      // Check if we're only re-centering the ensemble fcst around the det.
+      bool recenterOnly(false);
+      if ( fullConfig.has("recentering around deterministic") ) {
+        fullConfig.get("recentering around deterministic", recenterOnly);
+      }
```

Comment:
Nitpick: something like
```c++
bool recenterOnly = fullConfig.getBool("recentering around deterministic", false);
```
is a bit shorter. No need to change this though.

Reply:
looks cleaner, will change.